<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptimisationOptionsDialog</class>
 <widget class="QDialog" name="OptimisationOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Beat Optimisation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="automaticOptimisationRadioButton">
     <property name="text">
      <string>Automatic Optimisation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="AutomaticOptimisationOptionsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="roundBPMCheckBox">
        <property name="text">
         <string>Round automatically detected
BPM to nearest whole number</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="roundBPMEvenCheckBox">
        <property name="text">
         <string>Round to nearest even number</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="useProvidedOutputTempoRadioButton">
     <property name="text">
      <string>Use Provided Output Tempo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="manualOptimisationOptionsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="manualOptimisationOptionsLayout">
        <item>
         <layout class="QHBoxLayout" name="manualBPMLayout">
          <item>
           <widget class="QLabel" name="manualBPMLabel">
            <property name="text">
             <string>BPM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="manualBPMSpinBox">
            <property name="minimum">
             <double>15.000000000000000</double>
            </property>
            <property name="maximum">
             <double>600.000000000000000</double>
            </property>
            <property name="value">
             <double>120.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="manualStartTimestampCheckBox">
          <property name="text">
           <string>Use provided starting timestamp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="manualStartTimestampSpinBox">
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>OptimisationOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>OptimisationOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
